packages/npm-packages/ruby-wasm-wasi/test-e2e/support.ts

```diff
@@ -23,8 +24,33 @@ export const setupProxy = (context: BrowserContext) => {
     const request = route.request();
     console.log(">> [MOCK]", request.method(), request.url());
     const relativePath = request.url().match(cdnPattern)[1];
-    route.fulfill({
-      path: path.join(process.env.RUBY_NPM_PACKAGE_ROOT, "dist", relativePath),
-    });
+    const mockedPath = path.join(
+      process.env.RUBY_NPM_PACKAGE_ROOT,
+      "dist",
+      relativePath,
+    );
+
+    if (relativePath.match("error_on_load_twice.rb")) {
+      route.fulfill({
+        body: `raise 'load twice' if defined? ALREADY_LOADED
+               ALREADY_LOADED = true`,
+        contentType: "text/ruby",
+      });
```

Comment:
TODO: I feel uncomfortable writing responses for individual tests in this file that is shared by multiple tests.
I would like to separate them into separate files.